channels/prerelease-4.1.yaml

```diff
@@ -36,3 +36,4 @@ versions:
 - 4.1.28
 - 4.1.29
 - 4.1.30
+- 4.1.31
```

Comment:
https://github.com/openshift/cincinnati-graph-data/pull/27 will handle this individually